### ISSUE TITLE
[BUG] UWP appinstaller dependency packages wrong case for ARM packages

### ISSUE THREAD

**Issue by nick-goloborodko** [microsoft/msix-packaging#153]
*Project*
MSIX SDK or Win7Msix

**Describe the bug**
When generating appisntaller packaged from Visual Studio 2017 for a UWP app, some of the ARM dependency packages are using "ARM" folder in path rather than "arm" for the dependency packages. This causes issues when trying to use Azure Static Websites for distribution of the app, as it is case-sensitive.

Note: "arm" is correctly lower case on lines 8 and 11, and incorrectly uppercase "ARM" on lines 12 and 15

**To Reproduce**

**Expected behavior**

**Screenshots**
Sample appinstaller file generated:

```xml
<?xml version="1.0" encoding="utf-8"?>
<AppInstaller Uri="http://example.com/MyCompany.RFID.Client.App.appinstaller" Version="1.0.16.0" xmlns="http://schemas.microsoft.com/appx/appinstaller/2017/2">
  <MainBundle Name="5CD25A6F-36ED-485B-8D99-5C997ACEEC40" Version="1.0.16.0" Publisher="CN=nick.goloborodko" Uri="http://example.com/MyCompany.RFID.Client.App_1.0.16.0_Debug_Test/MyCompany.RFID.Client.App_1.0.16.0_x86_x64_arm_Debug.appxbundle" />
  <Dependencies>
    <Package Name="Microsoft.UI.Xaml.2.1" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="2.11904.5004.0" ProcessorArchitecture="arm" Uri="http://example.com/MyCompany.RFID.Client.App_1.0.16.0_Debug_Test/Dependencies/arm/Microsoft.UI.Xaml.2.1.appx" />
    <Package Name="Microsoft.NET.CoreRuntime.2.2" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="2.2.27404.2" ProcessorArchitecture="x86" Uri="http://example.com/MyCompany.RFID.Client.App_1.0.16.0_Debug_Test/Dependencies/x86/Microsoft.NET.CoreRuntime.2.2.appx" />
    <Package Name="Microsoft.NET.CoreRuntime.2.2" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="2.2.27404.2" ProcessorArchitecture="x64" Uri="http://example.com/MyCompany.RFID.Client.App_1.0.16.0_Debug_Test/Dependencies/x64/Microsoft.NET.CoreRuntime.2.2.appx" />
    <Package Name="Microsoft.NET.CoreRuntime.2.2" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="2.2.27404.2" ProcessorArchitecture="arm" Uri="http://example.com/MyCompany.RFID.Client.App_1.0.16.0_Debug_Test/Dependencies/arm/Microsoft.NET.CoreRuntime.2.2.appx" />
    <Package Name="Microsoft.NET.CoreFramework.Debug.2.2" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="2.2.27505.2" ProcessorArchitecture="x86" Uri="http://example.com/MyCompany.RFID.Client.App_1.0.16.0_Debug_Test/Dependencies/x86/Microsoft.NET.CoreFramework.Debug.2.2.appx" />
    <Package Name="Microsoft.NET.CoreFramework.Debug.2.2" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="2.2.27505.2" ProcessorArchitecture="x64" Uri="http://example.com/MyCompany.RFID.Client.App_1.0.16.0_Debug_Test/Dependencies/x64/Microsoft.NET.CoreFramework.Debug.2.2.appx" />
    <Package Name="Microsoft.NET.CoreFramework.Debug.2.2" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="2.2.27505.2" ProcessorArchitecture="arm" Uri="http://example.com/MyCompany.RFID.Client.App_1.0.16.0_Debug_Test/Dependencies/arm/Microsoft.NET.CoreFramework.Debug.2.2.appx" />
    <Package Name="Microsoft.VCLibs.140.00.Debug" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="14.0.27023.1" ProcessorArchitecture="arm" Uri="http://example.com/MyCompany.RFID.Client.App_1.0.16.0_Debug_Test/Dependencies/ARM/Microsoft.VCLibs.ARM.Debug.14.00.appx" />
    <Package Name="Microsoft.VCLibs.140.00.Debug" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="14.0.27023.1" ProcessorArchitecture="x64" Uri="http://example.com/MyCompany.RFID.Client.App_1.0.16.0_Debug_Test/Dependencies/x64/Microsoft.VCLibs.x64.Debug.14.00.appx" />
    <Package Name="Microsoft.VCLibs.140.00.Debug" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="14.0.27023.1" ProcessorArchitecture="x86" Uri="http://example.com/MyCompany.RFID.Client.App_1.0.16.0_Debug_Test/Dependencies/x86/Microsoft.VCLibs.x86.Debug.14.00.appx" />
    <Package Name="Microsoft.Services.Store.Engagement" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="10.0.1610.0" ProcessorArchitecture="arm" Uri="http://example.com/MyCompany.RFID.Client.App_1.0.16.0_Debug_Test/Dependencies/ARM/Microsoft.Services.Store.Engagement.ARM.10.0.appx" />
    <Package Name="Microsoft.Services.Store.Engagement" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="10.0.1610.0" ProcessorArchitecture="x64" Uri="http://example.com/MyCompany.RFID.Client.App_1.0.16.0_Debug_Test/Dependencies/x64/Microsoft.Services.Store.Engagement.x64.10.0.appx" />
    <Package Name="Microsoft.Services.Store.Engagement" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="10.0.1610.0" ProcessorArchitecture="x86" Uri="http://example.com/MyCompany.RFID.Client.App_1.0.16.0_Debug_Test/Dependencies/x86/Microsoft.Services.Store.Engagement.x86.10.0.appx" />
  </Dependencies>
  <UpdateSettings>
    <OnLaunch HoursBetweenUpdateChecks="0" />
  </UpdateSettings>
</AppInstaller>
```

**Platform**

**Additional context**

**Comment by msftrubengu** [comment]
Closing this issue as is not related to the MSIX SDK or MsixCore. I will try to redirect internally to the proper owners.

In the future, please use the [Feedback Hub App](https://docs.microsoft.com/en-us/windows-insider/feedback-hub/feedback-hub-app#add-new-feedback) or [Visual Studio Developer Community](https://developercommunity.visualstudio.com/spaces/8/index.html) to report it.

**Comment by edwardskrod** [comment]
@nick-goloborodko   
Nick,
This issue was resolved in Visual Studio 16.2 released on July 23.   Please let me know if you are still having issues.

**Comment by tunbeo** [comment]
I'm using Microsoft Visual Studio Community 2019 Version 16.5.3 and still have the problem
